### PR TITLE
Create copy of `KeyEvents` timeline cards 

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

Creates a copy of the [KeyEvents](https://github.com/guardian/dotcom-rendering/blob/428f0e93ac8f16f0843a641453331a41a002fee8/common-rendering/src/components/keyEvents.tsx) component.

## Why

As part of the work we're doing to create a key events carousel we need to significantly amend the key events component.

We're unable to incrementally make adjustments to the existing component as it needs to exist in its current form until the new carousel is ready to launch.

The PR is an exact copy of the existing `KeyEvent` component with no changes. Once merged it will give us a platform on which to make amendments and, and mean that any merge diffs will only show actual changes. 

With this approach we hope be able to work incrementally and avoid the risk of stale 'mega' branches, or 'big bang' releases.

### CODE REVIEW

**Review by oliverlloyd — CHANGES_REQUESTED** [review]
Apols for the red but can we create this component in `dotcom-rendering`, please?

**Comment by oliverlloyd** [comment]
> Apols for the red but can we create this component in `dotcom-rendering`, please?

For context, we've agreed that moving forward we won't be using `common-rendering` and will slowly migrate content away from it. All new files should be created locally.

See: https://docs.google.com/document/d/1O7Uc4mgAeddlw9ig_BMHfmual2LghQnAhvnbQ_nwB_M/edit#heading=h.xqk4jcq9wkgi

**Comment by oliverlloyd** [comment]
I would say that if you have a new requirement to build a carousel of the key events then there's probably very little that you want to take from the existing `keyEvents` components and it will likely be easier for you to start from scratch. Especially as so many of the code patterns and idioms in DCR are so different and you don't want us all over your PRs raising subjective nitpicks!

**Comment by joecowton1** [comment]
> > Apols for the red but can we create this component in `dotcom-rendering`, please?
> 
> For context, we've agreed that moving forward we won't be using `common-rendering` and will slowly migrate content away from it. All new files should be created locally.
> 
> See: https://docs.google.com/document/d/1O7Uc4mgAeddlw9ig_BMHfmual2LghQnAhvnbQ_nwB_M/edit#heading=h.xqk4jcq9wkgi

Thanks Oliver, I hadn't seen that before! It's an interesting discussion too. Will move it now.

**Comment by joecowton1** [comment]
> I would say that if you have a new requirement to build a carousel of the key events then there's probably very little that you want to take from the existing `keyEvents` components and it will likely be easier for you to start from scratch. Especially as so many of the code patterns and idioms in DCR are so different and you don't want us all over your PRs raising subjective nitpicks!

Haha, interestingly at the card / key event level at least there's a lot of crossover, or at least it seems that way for now. Let's see how it develops :)

**Comment by joecowton1** [comment]
@oliverlloyd moved to DCR now.

File: dotcom-rendering/src/web/components/KeyEventsCards.tsx [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+	from,
+} from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import {
+	ArticleFormat,
+	ArticlePillar,
+	ArticleTheme,
+	timeAgo,
+} from '@guardian/libs';
+import { darkModeCss } from '@guardian/common-rendering/src/lib';
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/components/KeyEventsCards.tsx:24]: This should not be imported. We don't use this in DCR

File: dotcom-rendering/src/web/components/KeyEventsCards.tsx [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+import {
+	textSans,
+	headline,
+	sport,
+	culture,
+	lifestyle,
+	opinion,
+	news,
+	neutral,
+	remSpace,
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/components/KeyEventsCards.tsx:14]: This should not be used in DCR

File: dotcom-rendering/src/web/components/KeyEventsCards.tsx [hunk 3]
```diff
@@ -0,0 +1,268 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/components/KeyEventsCards.tsx:4]: We don't use this in DCR

File: dotcom-rendering/src/web/components/KeyEventsCards.tsx [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+	ArticlePillar,
+	ArticleTheme,
+	timeAgo,
+} from '@guardian/libs';
+import { darkModeCss } from '@guardian/common-rendering/src/lib';
+import Accordion from '@guardian/common-rendering/src/components/accordion';
+import {
+	background,
+	text,
+} from '@guardian/common-rendering/src/editorialPalette';
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/components/KeyEventsCards.tsx:29]: This is the wrong palette to use for DCR

**Pull request closed without merging**